builds/misc/images-release.yaml

```diff
@@ -326,6 +326,31 @@ jobs:
         inputs:
           PathtoPublish: '$(Build.BinariesDirectory)/publish'
           ArtifactName: 'publish-linux'
+################################################################################
+  - job: linux_API_proxy_module
+################################################################################
+    timeoutInMinutes: 80
+    displayName: linuxAPIProxy
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+      - bash: |
+            docker login '$(build.registry.address)' --username '$(build.registry.user)' --password '$(build.registry.password)'
+            docker login '$(registry.address)' --username '$(registry.user)' --password '$(registry.password)'
+        displayName: 'Docker Login'
+      # Build API Proxy executable
+      - template: templates/build-api-proxy.yaml
```

Comment:
if the build artifact is executable, i think you need to add code sign step as well.

Reply:
I added bear to the PR. But when discussing with him it looked ok.

Reply:
Email to Varun to check if code sign for api proxy executable is required.